src/go/rpk/pkg/cli/cloud/byoc/install.go

```diff
@@ -60,10 +60,11 @@ autocompletion, start a new terminal and tab complete through it!
 		},
 	}
 	cmd.Flags().StringVar(&redpandaID, flagRedpandaID, "", flagRedpandaIDDesc)
+	cmd.MarkFlagRequired(flagRedpandaID)
 	return cmd
 }
 
-func loginAndEnsurePluginVersion(ctx context.Context, fs afero.Fs, cfg *config.Config, redpandaID string) (binPath string, token string, installed bool, rerr error) {
+func loginAndEnsurePluginVersion(ctx context.Context, fs afero.Fs, cfg *config.Config, redpandaID string, latest bool) (binPath string, token string, installed bool, rerr error) {
```

Comment:
tiny nit: I'd maybe name this `isLatest` for consistency with the rest of the PR